pkg/reconciler/events/cloudevent/cloudevent.go

```diff
@@ -35,12 +35,32 @@ import (
 type TektonEventType string
 
 const (
+	// TektonTaskRunStartedV1 is sent for TaskRuns with "ConditionSucceeded" "Unknown"
+	// the first time they are picked up by the reconciler
+	TektonTaskRunStartedV1 TektonEventType = "dev.tekton.event.taskrun.started.v1"
```

Comment:
NIT: rename `TektonTaskRunStarted` to `EventTaskRunStarted` 🤔  and same for all events below 😲

Reply:
Uhm, maybe `TaskRunStartedEvent`?

Reply:
Good point, that Tekton in there is redundant